pkg/chunk/cache/redis_cache.go

```diff
@@ -141,14 +141,3 @@ func (c *RedisCache) ping(ctx context.Context) error {
 	}
 	return err
 }
-
-func redisStatusCode(err error) string {
```

Comment:
@dmitsh do we want to use this?
There could be a field on `instrumentedCache` to call either this function or `instr.ErrorCode()` for Redis or Memcached.
But I'm not 100% sure we need to decode `ErrNil` now.